h2o-r/h2o-package/R/frame.R

```diff
@@ -3794,6 +3794,30 @@ h2o.tolower <- function(x) .newExpr("tolower", x)
 h2o.toupper <- function(x) .newExpr("toupper", x)
 
 #'
+#' Searches for matches to argument ‘pattern’ within each element
+#  of a string column.
+#'
+#' This function has similar semantics as R's native grep function
+#' and it supports a subset of its parameters. Default behavior is
```

Comment:
Is that necessary? the parameters of the R function are listed and the user can consult the list, the point of this statement was to make it clear that not everything is supported